### PR TITLE
docs: normalize CI list

### LINKED ISSUE #531: README inconsistent in the listing of supported CI platforms
1. "currently it supports GitHub Actions, Azure Pipelines, Travis CI, AppVeyor, and CircleCI"
2. "Builds manylinux, macOS and Windows wheels for CPython and PyPy using Azure Pipelines, Travis CI, AppVeyor, and CircleCI"
3. "cibuildwheel currently works on Travis CI, Azure Pipelines, AppVeyor, GitHub Actions, CircleCI, and Gitlab CI"

I'm happy to submit a PR that either dedups all of these listings or at least makes them consistent, but since none of the lists agree with each other I'm not sure who is right. 😅

### PR DESCRIPTION
**Pull request by henryiii**
Closes #531

Ordering now is the same as well.

### CODE REVIEW

**Comment by YannickJadoul** [comment]
While we're at it, why not update the "What does it do?" table, as well?
Currently, it says 

> ² Only supported on Travis

for the non-Intel manylinux architectures.

Ah, but wait, yes. We weren't advertising that yet? :-)

**Review by YannickJadoul — APPROVED** [review]

**Comment by henryiii** [comment]
I assume we fix this shortly before 1.8.0, since it's common to see the current README when visiting the page?

@joerick if you see any listings here that could be removed, we do repeat ourselves quite a bit listing the supported CI systems. ;)

File: README.md [hunk 1]
```diff
@@ -31,21 +31,22 @@ What does it do?
...
+- Builds manylinux, macOS, and Windows wheels for CPython and PyPy
+- Works on GitHub Actions, Azure Pipelines, Travis CI, AppVeyor, CircleCI, and GitLab CI
 - Bundles shared library dependencies on Linux and macOS through [auditwheel](https://github.com/pypa/auditwheel) and [delocate](https://github.com/matthew-brett/delocate)
 - Runs the library test suite against the wheel-installed version of your library
 
 Usage
 -----
 
-`cibuildwheel` currently works on **Travis CI**, **Azure Pipelines**, **AppVeyor**, **GitHub Actions**, **CircleCI**, and **Gitlab CI**. Check the table below for supported platforms on each service:
+`cibuildwheel` currently works on **GitHub Actions**, **Azure Pipelines**, **Travis CI**, **AppVeyor**, **CircleCI**, and **GitLab CI**. Check the table below for supported platforms on each service:
```
Comment (joerick) [inline README.md:42]: This reference seems especially redundant, given the info is in the table below.
```suggestion
`cibuildwheel` runs inside a CI service. Supported platforms depend on which service you're using:
```

File: README.md [hunk 2]
```diff
@@ -92,7 +93,7 @@ jobs:
           path: ./wheelhouse/*.whl
 ```
 
-For more information, including building on Python 2, PyPI deployment, and the use of other CI services like Travis CI, Appveyor, Azure Pipelines, or CircleCI, check out the [documentation](https://cibuildwheel.readthedocs.org) and the [examples](https://github.com/joerick/cibuildwheel/tree/master/examples).
+For more information, including building on Python 2, PyPI deployment, and the use of other CI services like Azure Pipelines, Travis CI, AppVeyor, CircleCI, and GitLab CI, check out the [documentation](https://cibuildwheel.readthedocs.org) and the [examples](https://github.com/joerick/cibuildwheel/tree/master/examples).
```
Comment (joerick) [inline README.md:96]: We could also remove the CI service listing here, if you like?

**Review by joerick — COMMENTED** [review]
This looks good to me. I've noted a couple of places we could lose the listings :)

File: README.md (hunk 2 above)
Comment (henryiii) [reply README.md:96]:
```suggestion
For more information, including building on Python 2, PyPI deployment, and the use of other CI services, check out the [documentation](https://cibuildwheel.readthedocs.org) and the [examples](https://github.com/joerick/cibuildwheel/tree/master/examples).
```

**Comment by henryiii** [comment]
@joerick Any thoughts to enabling the merge when checks complete + required checks settings on GitHub?

**Comment by joerick** [comment]
weirdly, I tried enabling it and it gives this... no option to auto-merge...

image

**Comment by henryiii** [comment]
Did you go to the branch tab and enable required checks for the master branch? With no required checks, it has no idea what to consider "passing", as checks are added asynchronously at any time.

**Comment by henryiii** [comment]
You did check the box on the main settings page, too? (it was there?)

**Comment by joerick** [comment]
> You did check the box on the main settings page, too? (it was there?)

Ah, that's what I was missing. I think it's working ⬆️

**Pull request merged**